### ISSUE TITLE
An error occurs with Entities 0.50

### ISSUE THREAD

**Issue by kawai125** [hadashiA/VContainer#358]
The API for PlayerLoop of ECS `world`, `ScriptBehaviourUpdateOrder.UpdatePlayerLoop(World, Nullable<PlayerLoopSystem>)`, is removed from Entities 0.50, and that causes the error.

```
Library\PackageCache\jp.hadashikick.vcontainer@0749c2e7e2\Runtime\Unity\InstanceProviders\WorldInstanceProvider.cs(32,44): error CS0117: 'ScriptBehaviourUpdateOrder' does not contain a definition for 'UpdatePlayerLoop'
```

I guess it must be replaced as:

```
ScriptBehaviourUpdateOrder.RemoveWorldFromCurrentPlayerLoop(world);
ScriptBehaviourUpdateOrder.AddWorldToPlayerLoop(world, PlayerLoop.GetCurrentPlayerLoop());
```

The changelog @ [0.14.0] - 2020-08-04 says:

> Deprecated
> -  `ScriptBehaviourUpdateOrder.UpdatePlayerLoop()` is being deprecated in favor of `ScriptBehaviourUpdateOrder.AddWorldToPlayerLoop()`. Due to slightly different semantics, a direct automated API update is not possible: the new function always takes a `PlayerLoopSystem` object to modify, does not call `UnityEngine.LowLevel.PlayerLoop.SetPlayerLoop()`, and does not create the top-level system groups if they don't exist.

I'm sorry I don't know 'RemoveWorldFromCurrentPlayerLoop(world)' is required or not before calling 'AddWorldToPlayerLoop(world)'.

ref: [Class ScriptBehaviourUpdateOrder @ Entities 0.50.0-preview.24](https://docs.unity3d.com/Packages/com.unity.entities@0.50/api/Unity.Entities.ScriptBehaviourUpdateOrder.html)

**Comment by muveso** [comment]
the bug still exist.

**Comment by hadashiA** [comment]
Fixed #391. Thanks for the report and suggestion.
